TypeCobol.Test/Parser/Programs/Cobol2002/PublicGlobalOrPrivateGlobalTypedefs.rdzSYM.txt

```diff
@@ -0,0 +1,161 @@
+        Level: 5
+        IsFiller: False
+  Level: 1
+  IsFiller: False
+Programs:
+  Nested (.NET Type=ProgramSymbol, Kind=Program)
+  Owner: DVZFSCT0
+  Type:
+    (.NET Type=ProgramType, Tag=Program)
+  Types:
+    type3 (.NET Type=TypedefSymbol, Kind=Typedef)
+    Flags: [Public, Global, WORKING_STORAGE, InsideTypedef]
+    Owner: DVZFSCT0::Nested
+    Type:
+      (.NET Type=TypedefType, Tag=Typedef)
+      Flags: [InsideTypedef]
+      Symbol: DVZFSCT0::Nested::type3
+      TargetType:
+        (.NET Type=GroupType, Tag=Group)
+        Flags: [InsideTypedef]
+        Fields:
+          champ1 (.NET Type=VariableSymbol, Kind=Variable)
+          Flags: [Global, WORKING_STORAGE, InsideTypedef]
+          Owner: DVZFSCT0::Nested::type3
+          Type:
+            (.NET Type=PictureType, Tag=Picture)
+            Flags: [InsideTypedef]
+            Picture: X(1)
+          Level: 5
+          IsFiller: False
+    Level: 1
+    IsFiller: False
+    type4 (.NET Type=TypedefSymbol, Kind=Typedef)
+    Flags: [Private, Global, WORKING_STORAGE, InsideTypedef]
+    Owner: DVZFSCT0::Nested
+    Type:
+      (.NET Type=TypedefType, Tag=Typedef)
+      Flags: [InsideTypedef]
+      Symbol: DVZFSCT0::Nested::type4
+      TargetType:
+        (.NET Type=GroupType, Tag=Group)
+        Flags: [InsideTypedef]
+        Fields:
+          champ1 (.NET Type=VariableSymbol, Kind=Variable)
+          Flags: [Global, WORKING_STORAGE, InsideTypedef]
+          Owner: DVZFSCT0::Nested::type4
+          Type:
+            (.NET Type=PictureType, Tag=Picture)
+            Flags: [InsideTypedef]
+            Picture: X(1)
+          Level: 5
+          IsFiller: False
+    Level: 1
+    IsFiller: False
+  IsNested: True
+IsNested: False
+
+--- Program ---
+Stacked (.NET Type=ProgramSymbol, Kind=Program)
+Type:
+  (.NET Type=ProgramType, Tag=Program)
+Types:
+  type5 (.NET Type=TypedefSymbol, Kind=Typedef)
+  Flags: [Public, Global, WORKING_STORAGE, InsideTypedef]
+  Owner: Stacked
+  Type:
+    (.NET Type=TypedefType, Tag=Typedef)
+    Flags: [InsideTypedef]
+    Symbol: Stacked::type5
+    TargetType:
+      (.NET Type=GroupType, Tag=Group)
+      Flags: [InsideTypedef]
+      Fields:
+        champ1 (.NET Type=VariableSymbol, Kind=Variable)
+        Flags: [Global, WORKING_STORAGE, InsideTypedef]
+        Owner: Stacked::type5
+        Type:
+          (.NET Type=PictureType, Tag=Picture)
+          Flags: [InsideTypedef]
+          Picture: X(1)
+        Level: 5
+        IsFiller: False
+  Level: 1
+  IsFiller: False
+  type6 (.NET Type=TypedefSymbol, Kind=Typedef)
+  Flags: [Private, Global, WORKING_STORAGE, InsideTypedef]
+  Owner: Stacked
+  Type:
+    (.NET Type=TypedefType, Tag=Typedef)
+    Flags: [InsideTypedef]
+    Symbol: Stacked::type6
+    TargetType:
+      (.NET Type=GroupType, Tag=Group)
+      Flags: [InsideTypedef]
+      Fields:
+        champ1 (.NET Type=VariableSymbol, Kind=Variable)
+        Flags: [Global, WORKING_STORAGE, InsideTypedef]
```

Comment:
So here both type5::champ1 and type6::champ1 have the same Flags attributes, we don't see public or private. Maybe this should also be present here?

Reply:
Public and Private flags are for types and functions only so they should not be copied onto variable symbols. Besides this would not be used to check visibility of these data items because of expansion mechanism.